scripts/steps/build-api.js

```diff
@@ -110,6 +104,105 @@ const UNWRAP_UTILITIES = new Set([
   'Readonly'
 ])
 
+const EXCLUDED_SUBMENU_KINDS = new Set(['Type alias', 'Interface', 'Namespace'])
+const EXCLUDED_TREE_KINDS = new Set(['Namespace'])
+const CORE_ENTITIES = new Set([
+  'BaseNode',
+  'ClientNode',
+  'Connection',
+  'Log',
+  'LogStore',
+  'Reconnect',
+  'ServerConnection',
+  'ServerNode'
+])
+const CLIENT_ENTITIES = new Set([
+  'Client',
+  'CrossTabClient',
+  'badge',
+  'buildNewSyncMap',
+  'changeSyncMap',
+  'changeSyncMapById',
+  'confirm',
+  'createFilter',
+  'createSyncMap',
+  'defineSyncMap',
+  'deleteSyncMap',
+  'deleteSyncMapById',
+  'encryptActions',
+  'request'
+])
+const SERVER_ENTITIES = new Set([
+  'BaseServer',
+  'ChannelContext',
+  'Context',
+  'ResponseError',
+  'Server',
+  'ServerClient',
+  'del',
+  'get',
+  'patch',
+  'post',
+  'put',
+  'request'
+])
+const STATE_ENTITIES = new Set([
+  'cleanStores',
+  'createDerived',
+  'createMap',
+  'createPersistent',
+  'createRouter',
+  'createStore',
+  'getPagePath',
+  'getValue',
+  'openPage'
+])
+const TEST_ENTITIES = new Set([
+  'TestClient',
+  'TestLog',
+  'TestPair',
+  'TestServer',
+  'TestTime',
+  'emptyInTest',
+  'prepareForTest',
+  'eachStoreCheck'
+])
+
+const GROUPS = {
+  Client: 'Client',
+  Core: 'Core',
+  Other: 'Other',
+  React: 'React',
+  Server: 'Server',
+  State: 'Logux State',
+  Tests: 'Tests',
+  Vue: 'Vue'
+}
+
+// More specific group conditions should be placed higher
+const GROUPS_CONDITION = {
+  [GROUPS.React]: node => isSource(node, 'react'),
```

Comment:
We will support `Preact` soon. We need to add it behind `Vue` and fix check (since it put `preact` to `React` category)